.github/workflows/examples-datadog.yaml

```diff
@@ -0,0 +1,17 @@
+name: Deploy Module Examples - datadog
+
+# since is not needed every time and requires the datadog api and app keys this workflow is on dispatch
```

Comment:
I would add keys to our sandbox account like how lacework is. otherwise we'll get lazy and miss stuff

Reply:
Right, missed that comment during the cleanup, keys are already in SM, I will cleanup the workflow